src/controllers/group.controller.ts

```diff
@@ -9,8 +9,16 @@ export function getGroupByPk(req: express.Request, res: express.Response) {
     try {
       const id = req.params.id;
       const result = await service.getGroupByPk(id);
+      if (!result) {
+        throw 'Group could not found';
+      }
       res.json(result);
     } catch (error) {
+      console.error({
```

Comment:
I guess you should use logger instead of a direct console call

Reply:
It was in homework, some places they wanted to use direct console and other places logger. I do not understand why they want to mix, but anyway I was following the homework.